main.tf

```diff
@@ -108,7 +108,7 @@ resource "aws_kms_ciphertext" "grafana_password" {
 }
 
 resource "aws_kms_ciphertext" "thanos_password" {
-  count     = 1 - local.storage
+  count     = local.storage
```

Comment:
count based on string object? is it works?

Reply:
local.storage it's not the same that storage =)